src/main/java/com/anchore/jenkins/plugins/anchore/AnchoreBuilder.java

```diff
@@ -307,6 +317,7 @@ public DescriptorImpl getDescriptor() {
     // Default job level config that may be used both by config.jelly and an instance of AnchoreBuilder
     public static final String DEFAULT_NAME = "anchore_images";
     public static final String DEFAULT_ENGINE_RETRIES = "300";
+    public static final String DEFAULT_POLL_INTERVAL = "10";
```

Comment:
It looks like this would change the default from the previous, de-facto 1 seconds to 10 seconds. I think it would be preferable to leave that unchanged for existing users, but allow those who want a larger interval to configure that.

Reply:
Changed interval to 1 second as before.
https://github.com/brahankv/anchore-container-scanner-plugin/commit/9260bd27b8eacaf47db21cfa685806fb2cf3b3fb